ambry-cloud/src/main/java/com/github/ambry/cloud/AzureStorageTokenWriter.java

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright 2023 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package com.github.ambry.cloud;
+
+import com.github.ambry.cloud.azure.AzureCloudConfig;
+import com.github.ambry.clustermap.ClusterMap;
+import com.github.ambry.config.CloudConfig;
+import com.github.ambry.config.VerifiableProperties;
+import com.github.ambry.replication.FindTokenHelper;
+import com.github.ambry.replication.PartitionInfo;
+import com.github.ambry.replication.RemoteReplicaInfo;
+import com.github.ambry.replication.ReplicaTokenPersistor;
+import com.github.ambry.replication.ReplicationMetrics;
+import java.io.IOException;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+/**
+ * Implements methods to persist {@link com.github.ambry.replication.RemoteReplicaInfo.ReplicaTokenInfo} to Azure
+ */
+public class AzureStorageTokenWriter extends ReplicaTokenPersistor {
```

Comment:
What's our conclusion for the token? Do we use the existing abs for token or cosmos to save token?

Reply:
I will discuss this when we get to that PR but my focus now is to get this one out.

Reply:
I found that ABS has a table offering similar to cosmos 
https://learn.microsoft.com/en-us/azure/cosmos-db/table/support?toc=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fazure%2Fstorage%2Ftables%2Ftoc.json&bc=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fazure%2Fbread%2Ftoc.json